src/histolab/slide.py

```diff
@@ -205,9 +210,7 @@ def locate_biggest_tissue_box(
             PIL Image of the rescaled slide with the biggest tissue bounding box
             outlined.
         """
-        if not os.path.exists(self.scaled_image_path(scale_factor)):
-            self.save_scaled_image(scale_factor)
-        img = PIL.Image.open(self.scaled_image_path(scale_factor))
+        img = self.scaled_image(scale_factor)
```

Comment:
i don't like to replace `img` some line further, can we just pass `self.scaled_image(scale_factor)` to the filters at line 217?

Reply:
we need `img` defined before because here https://github.com/histolab/histolab/blob/refactor-slide-save-paths/src/histolab/slide.py#L220 you need to draw a line on it